src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -56,6 +56,24 @@ if not [%NET46_PUBLISH_ERROR_LEVEL%]==[0] (
 	exit /b %NET46_PUBLISH_ERROR_LEVEL%
 )
 
+set ILASM_PROJECTJSON=%BUILDTOOLS_PACKAGE_DIR%\ilasm\project.json
```

Comment:
I don't know exactly how coupled they are. I would imagine that there is some set of functionality that ilasm pulls from CoreCLR (and would need to be using the latest version of CoreCLR in order to work properly). Right now, we are able to run the latest ilasm on our 1.1 runtime, but I don't think it's a safe thing to rely on in the future.